docs/web-servers/lemp/how-to-install-the-lemp-stack-on-ubuntu-18-04/index.md

```diff
@@ -0,0 +1,238 @@
+
+    {{< file "/etc/nginx/sites-available/example.com.conf" nginx >}}
+server {
+    listen         80;
+    listen         [::]:80;
+    server_name    example.com www.example.com;
+    root           /var/www/html/example.com/public_html;
+    index          index.html;
+
+    location / {
+      try_files $uri $uri/ =404;
+    }
+
+    location ~* \.php$ {
+      fastcgi_pass unix:/run/php/php7.2-fpm.sock;
+      include         fastcgi_params;
+      fastcgi_param   SCRIPT_FILENAME    $document_root$fastcgi_script_name;
+      fastcgi_param   SCRIPT_NAME        $fastcgi_script_name;
+    }
+}
+{{< /file >}}
+
+    Here's a breakdown of the `server` block above:
+
+    -  NGINX is listening on port `80` for incoming connections to `example.com` or `www.example.com`.
+
+    -  The site is served out of `/var/www/html/example.com/public_html` and its index page (`index.html`) is a simple `.html` file. **If your index page will use PHP like WordPress does, substitute `index.html` for `index.php`.**
+
+    -  `try_files` tells NGINX to verify that a requested file or directory [actually exists](https://nginx.org/en/docs/http/ngx_http_core_module.html#try_files) in the site's root filesystem before further processing the request. If it does not, a `404` is returned.
+
+    -  `location ~* \.php$` means that NGINX will apply this configuration to all .php files (file names are not case sensitive) in your site’s root directory, including any subdirectories containing PHP files.
+
+    -  The `*` in the `~* \.php$` location directive indicates that PHP file names are not case sensitive. This can be removed if you prefer to enforce letter case.
+
+    -  `fastcgi_pass` specifies the [UNIX socket](https://nginx.org/en/docs/http/ngx_http_fastcgi_module.html#fastcgi_pass) where PHP listens for incoming connections from other local processes
+
+    -  `include fastcgi_params` tells NGINX to process a list of `fastcgi_param` variables at `/etc/nginx/fastcgi_params`.
+
+    -  The `fastcgi_param` directives contain the [location](https://nginx.org/en/docs/http/ngx_http_fastcgi_module.html#variables) (relative to the site's root directory) and file [naming convention](https://nginx.org/en/docs/http/ngx_http_fastcgi_module.html#fastcgi_index) of PHP scripts to be served when called by NGINX.
+
+4.  Create a link to your website configuration file from within the sites-enabled directory. Change the name of the file to the name you used for your domain:
+
+        sudo ln -s /etc/nginx/sites-available/example.com.conf /etc/nginx/sites-enabled/
+
+5.  Enable the firewall to allow web traffic. This guide lists the commands to enable web traffic if you configured UFW on your server.
+
+      a. Check the ports that are enabled for `Nginx Full` Profile:
+
+        sudo ufw app info "Nginx Full"
+
+      Ports `80` and `443` should be listed as enabled for `Nginx Full` profile.
+
+      b. To allow incoming HTTP and HTTPS traffic for `Nginx Full` profile:
```

Comment:
As previously mentioned, why the a. and b. ? Sentence should also read "if these ports are not allowed, enable them..." or something similar.

Reply:
restructured this to be a new section